org.eclipse.jdt.core/codeassist/org/eclipse/jdt/internal/codeassist/DOMCompletionEngine.java

```diff
@@ -3318,4 +3351,11 @@ private ITypeBinding getDeclaringClass(IBinding binding) {
 			binding instanceof IVariableBinding variableBinding && variableBinding.isField() ? variableBinding.getDeclaringClass() :
 			null;
 	}
+
+	int computeRelevanceForInterestingProposal(){
+		return RelevanceConstants.R_INTERESTING;
+	}
+	int computeRelevanceForResolution(){
+		return RelevanceConstants.R_RESOLVED;
+	}
```

Comment:
I really don't see the point of making these methods; I honestly think that we should inline the constants where possible. However, that can probably wait for another commit.